src/Symfony/Component/Serializer/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ CHANGELOG
 ---
 
  * Add `XmlEncoder::SAVE_OPTIONS` context option
+ * Deprecate the `MissingConstructorArgumentsException` class
```

Comment:
let's put this on one line:
  * Deprecate `MissingConstructorArgumentsException` in favor of `MissingConstructorArgumentException`

Reply:
Done